app/src/main/AndroidManifest.xml

```diff
@@ -243,8 +243,10 @@
             </intent-filter>
         </activity>
 
+        <!-- we handle orientation configChanges to ensure proper tab UNDO functionality -->
         <activity
             android:name="com.duckduckgo.app.tabs.ui.TabSwitcherActivity"
+            android:configChanges="orientation"
```

Comment:
This is done to ensure proper handing of configuration changes vs just navigating back
* when we navigate back we want to clean the "deletable" tabs
* when tab screen rotates we do not want to clean "deletable" tabs

I didn't find any other way to do this logic. Anybody has a simple solution that does not require changes in manifest?

Reply:
I removed this from the manifest, found a [better way to do it](https://github.com/duckduckgo/Android/pull/1011/commits/b33a9213ffaa2287263823d358d5efd478656249)
Will add the test to the PR description.

Let me know if I can resolve this comment